force-app/main/default/classes/DuplicateEnrollmentTriggerTest.cls

```diff
@@ -0,0 +1,76 @@
+@isTest
+public class DuplicateEnrollmentTriggerTest 
+{
+    @TestSetup
+    static void makeData()
+    {
+        SchoolDataCreator.createData();
+        Student__c s = [SELECT Id FROM Student__c LIMIT 1];
+        Class__c c = [SELECT Id FROM Class__c LIMIT 1];
+        ClassEnrollment__c ce = new ClassEnrollment__c();
+        ce.Student__c = s.Id;
+        ce.Class__c = c.Id;
+        ce.Credits__c = 1;
+        insert ce;
+    }  
+
+    @isTest
+    public static void testSingleInsert()
```

Comment:
Your test names should include the methods in which you are testing. It's hard to know what this is for without context.